client/performance-profiler/components/metric-tab-bar/style.scss

```diff
@@ -24,33 +24,18 @@ $blueberry-color: #3858e9;
 	padding: 16px 22px 16px 22px;
 	width: 100%;
 	border: 1px solid transparent;
-	&:not(.active) {
-		border-bottom: 1px solid var(--gutenberg-gray-100, #f0f0f0);
 
-		&:not(:nth-child(-n+2)) {
-			border-top-color: transparent;
-		}
-
-		&:last-child {
-			border-bottom-color: transparent;
-		}
-
-		&.metric-tab-bar__performance {
-			border-bottom-color: transparent;
-		}
-
-	}
+	border-bottom: 1px solid var(--gutenberg-gray-100, #f0f0f0);
 
 	&.active {
 		/* stylelint-disable-next-line scales/radii */
 		border-radius: 6px;
-		border-color: transparent;
 		outline: 1.5px solid $blueberry-color;
-		position: relative;
+		outline-offset: -0.5px;
```

Comment:
OMG negative `outline-offset` saves the day! 😍

Reply:
What the heck?? I didn't even know it's possible!